internal/lightcone/preservation/trendoftheuniversalmarket/trendoftheuniversalmarket.go

```diff
@@ -0,0 +1,73 @@
+package trendoftheuniversalmarket
+
+import (
+	"github.com/simimpact/srsim/internal/global/common"
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	mod    key.Modifier = "trend"
+	moddot key.Modifier = "trend-burn"
```

Comment:
nit: `trend` may not be a clear name to folks when reading the logs (though I am fine with it, can be someone else's PR if they complain)